packages/js/vite.config.js

```
@@ -0,0 +1,74 @@
+import { defineConfig } from "vite";
+import path from "path";
+
+export default defineConfig(({ mode }) => {
```

ESLint

[error] 'mode' is defined but never used.

(no-unused-vars)